lib/manager/common.ts

```diff
@@ -173,15 +173,17 @@ export interface ManagerApi {
   extractAllPackageFiles?(
     config: ExtractConfig,
     files: string[]
-  ): Result<PackageFile[]>;
+  ): Result<PackageFile[] | null>;
 
   extractPackageFile?(
     content: string,
     packageFile?: string,
     config?: ExtractConfig
-  ): Result<PackageFile>;
+  ): Result<PackageFile | null>;
 
-  getPackageUpdates(config: PackageUpdateConfig): Result<PackageUpdateResult[]>;
+  getPackageUpdates?(
```

Comment:
@rarkins Is `getPackageUpdates` optional?

Reply:
Yes, I think so

Reply:
Only `lib/manager/travis` implements this.